### PR TITLE
bee 1.5.3 (new formula)

### PR DESCRIPTION
**Pull request by danielfariati**
Bee is a database changes manager: https://github.com/bluesoft/bee

### CODE REVIEW

**Comment by danielfariati** [comment]
Apparently, a github issue:
> Error: GitHub API rate limit exceeded for 208.78.110.199. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)

**Comment by DomT4** [comment]
Feel free to ignore that. Won't happen on Jenkins, which is the more important CI for now.

File: Library/Formula/bee.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  test do
+    (testpath/"bee.properties").write <<-EOS.undent
+      test-database.driver=com.mysql.jdbc.Driver
+      test-database.url=jdbc:mysql://127.0.0.1/test-database
+      test-database.user=root
+      test-database.password=
+    EOS
+    mkdir "#{testpath}/bee"
+    system "#{libexec}/bin/bee", "dbchange:create new-file"
```
Comment (DomT4) [inline Library/Formula/bee.rb]: Can we use the `bin/"bee"` symlink here rather than the direct `libexec` path?

File: Library/Formula/bee.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  end
+
+  test do
+    (testpath/"bee.properties").write <<-EOS.undent
+      test-database.driver=com.mysql.jdbc.Driver
+      test-database.url=jdbc:mysql://127.0.0.1/test-database
+      test-database.user=root
+      test-database.password=
+    EOS
+    mkdir "#{testpath}/bee"
```
Comment (DomT4) [inline Library/Formula/bee.rb]: It's a nit, but `(testpath/"bee").mkpath` looks a bit nicer.

**Comment by danielfariati** [comment]
Changes commited, @DomT4.
The code looks so much better now!
Thanks for the tips.

**Comment by danielfariati** [comment]
Had to rollback some changes. Hope it passes now.

**Comment by danielfariati** [comment]
Finally, it passed ;)

**Comment by DomT4** [comment]
It looks like `https://github.com/bluesoft/bee/releases/download/1.54/bee-1.54.zip` is available now, just to be annoying. If you push the update I'll merge this once that passes.

**Comment by danielfariati** [comment]
Sorry about the delay.
Updated.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/3c8f75e10f2eb33391a13ce8a7b3ed4538c8c5bb. Thank you for your contribution to Homebrew @danielfariati; we appreciate it! :smiley_cat:

**Pull request closed without merging**